core/templates/pages/exploration-editor-page/statistics-tab/issues/early-quit-issue.component.html

```diff
@@ -21,10 +21,10 @@
 </div>
 <div class="text-center">
   <div class="row">
-    <div ng-repeat="playthroughId in playthroughIds" class="col-md-4" style="margin-left: -40px;">
+    <div ng-repeat="playthroughId in playthroughIds" class="col-md-4 oppia-playthrough-id">
```

Comment:
This isn't actually representing a "playthrough id" -- please call the class something like oppia-playthrough-container.

Reply:
+1